### PR TITLE
Change header if device is not discoverable

### LINKED ISSUE #1: Claims to be discoverable when bluetooth is disabled
We need to figure out the adapter header copy for when the adapter is disabled

Launchpad Details: [#LP1650937](https://bugs.launchpad.net/bugs/1650937) Daniel Fore - 2016-12-18 19:19:49 +0000

### PR DESCRIPTION
**Pull request by DDieter**
Checks the discoverable property of the bluetooth adapter and modifies the header accordingly.

Fixes #1.

### CODE REVIEW

File: src/HeaderAdapter.vala [hunk 1]
```diff
@@ -66,4 +73,13 @@ public class Bluetooth.HeaderAdapter : Gtk.Grid {
             }
         });
     }
+
+    private void update_label () {
+        critical (adapter.discoverable.to_string ());
```
Comment (donadigo) [inline src/HeaderAdapter.vala]: Please use `info ()` instead of `critical ()`.

Comment (DDieter) [reply src/HeaderAdapter.vala]: Something I forgot to remove after testing. Changed it to info now.

**Comment by DDieter** [comment]
@matzipan This seems to be an unrelated issue, because my phone actually can't find the device after turning bluetooth off and on again.

**Comment by jeremypw** [comment]
I have updated this PR to be compatible with changes in master and work.

File: src/Services/Manager.vala [hunk 2]
```diff
@@ -98,14 +99,6 @@ public class Bluetooth.Services.ObjectManager : Object {
                                 adapter.stop_discovery.begin ();
                             }
                         }
-                        
-                    }
-
-                    var discoverable = changed.lookup_value("Discoverable", new VariantType("b"));
```
Comment (danrabbit) [inline src/Services/Manager.vala]: Is there a reason this was removed? This looks like the mechanism that ensures that we remain discoverable while the plug is open

Comment (jeremypw) [reply src/Services/Manager.vala]: As far as I could see it does nothing.  It appears to be saying if the value of adapter.discoverable changes then ensure that adapter.discoverable is what it was signalled to have changed to anyway??

**Pull request merged**